<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DelWindow</class>
 <widget class="QMainWindow" name="DelWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>659</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1021</width>
    <height>659</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1021</width>
    <height>659</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Delete - Cataloging Department </string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Эмблема2.png</normaloff>Эмблема2.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lab_pic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1021</width>
      <height>881</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>lib1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="lab_nln">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(88, 53, 59);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_back">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>70</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 228, 224);
color: rgb(86, 54, 57);</string>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_naz_del">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>120</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(86, 54, 57);
color: rgb(229, 228, 224);</string>
    </property>
    <property name="text">
     <string>Удаление</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>1001</width>
      <height>511</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(198, 183, 178);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="keep">
     <attribute name="title">
      <string>Хранение</string>
     </attribute>
     <widget class="QTableView" name="tab_keep">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>951</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_nazv_k">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>280</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 254, 250);
color: rgb(77, 61, 58);</string>
      </property>
      <property name="text">
       <string>Хранение</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lab_naz_mark_k">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>320</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 254, 250);
color: rgb(77, 61, 58);</string>
      </property>
      <property name="text">
       <string>  Авторский знак:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cB_mark_k">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>340</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(88, 53, 59);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_del_k">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>320</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(247, 246, 243);
color: rgb(81, 50, 55);</string>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="publish">
     <attribute name="title">
      <string>Издательства</string>
     </attribute>
     <widget class="QTableView" name="tab_publish">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>951</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_nazv_p">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>280</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 254, 250);
color: rgb(77, 61, 58);</string>
      </property>
      <property name="text">
       <string>Издательства</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lab_naz_name_p">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>320</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 254, 250);
color: rgb(77, 61, 58);</string>
      </property>
      <property name="text">
       <string>  Название:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cB_name_p">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>340</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(88, 53, 59);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_del_p">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>320</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(247, 246, 243);
color: rgb(81, 50, 55);</string>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="book">
     <attribute name="title">
      <string>Книги</string>
     </attribute>
     <widget class="QTableView" name="tab_book">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>951</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_nazv_b">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>280</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 254, 250);
color: rgb(77, 61, 58);</string>
      </property>
      <property name="text">
       <string>Книги</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lab_naz_name_b">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>320</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 254, 250);
color: rgb(77, 61, 58);</string>
      </property>
      <property name="text">
       <string>  Название:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_naz_isbn_b">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>320</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 254, 250);
color: rgb(77, 61, 58);</string>
      </property>
      <property name="text">
       <string>  ISBN:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_del_b">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>320</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(247, 246, 243);
color: rgb(81, 50, 55);</string>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cB_name_b">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>340</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(88, 53, 59);</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="te_isbn_b">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>340</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(88, 53, 59);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="order">
     <attribute name="title">
      <string>Заказ книги</string>
     </attribute>
     <widget class="QTableView" name="tab_order">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>951</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_nazv_o">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>280</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 254, 250);
color: rgb(77, 61, 58);</string>
      </property>
      <property name="text">
       <string>Заказ книги</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lab_naz_nameb_o">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>320</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 254, 250);
color: rgb(77, 61, 58);</string>
      </property>
      <property name="text">
       <string>  Название книги:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cB_nameb_o">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>340</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(88, 53, 59);</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_naz_read_o">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>320</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 254, 250);
color: rgb(77, 61, 58);</string>
      </property>
      <property name="text">
       <string>  Читатель:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cB_read_o">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>340</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(88, 53, 59);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_del_o">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>320</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(247, 246, 243);
color: rgb(81, 50, 55);</string>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="reader">
     <attribute name="title">
      <string>Читатели</string>
     </attribute>
     <widget class="QTableView" name="tab_read">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>951</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_nazv_r">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>280</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 254, 250);
color: rgb(77, 61, 58);</string>
      </property>
      <property name="text">
       <string>Читатели</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="cB_lname_r">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>340</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(88, 53, 59);</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_naz_lname_r">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>320</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 254, 250);
color: rgb(77, 61, 58);</string>
      </property>
      <property name="text">
       <string>  Фамилия читателя:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_del_r">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>320</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(247, 246, 243);
color: rgb(81, 50, 55);</string>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
